README.md

```diff
@@ -337,6 +337,60 @@ thread apply all bt
 
 This will dump all the threads stack traces into gdb.txt
 
+
+Testnet Debugging
+---
+
+We maintain several testnets:
+
+* `testnet` - public stable channel testnet accessible via testnet.solana.com. Runs 24/7
```

Comment:
These are both beta channel nets, not sure why we'd document them as being on the stable channel until they really are.

Reply:
I changed it back to beta. Not sure why we aren't pointing those to stable, but we can discuss that outside this PR.